source/PokerSetup.py

```diff
@@ -60,12 +60,12 @@ def preGame(self, players: PlayerList):
 
         return Big_Blind
 
-    def Blinds(self):
-        #   Determines who is Big_Blind and who is Small_Blind.
-        #   Made a BlindsList to keep track.
-        #   BlindList will need to delete all players no longer playing.
-        #   Made a PlayersOutOfGameList, just keep the BlindList updated every time by deleting from PlayersOutOfGameList.
-        pass
+    def Blinds(self, playeringamelist, round):
+        Dealer = playeringamelist[round]
+        Small_Blind = playeringamelist[round+1]
```

Comment:
consider using a modulus to ensure there are no index out of bounds errors when you are adding.